spec/models/article_spec.rb

```diff
@@ -3,12 +3,16 @@
     it { is_expected.to have_db_column :title }
     it { is_expected.to have_db_column :lede }
     it { is_expected.to have_db_column :body }
+    it { is_expected.to have_db_column :category_id}
+    it { is_expected.to have_db_column :category_name}
   end
 
   describe 'Validations' do
     it { is_expected.to validate_presence_of :title }
     it { is_expected.to validate_presence_of :lede }
     it { is_expected.to validate_presence_of :body }
+    it { is_expected.to validate_presence_of :category_id }
+    it { is_expected.to validate_presence_of :category_name }
   end
 
   describe 'Factory' do
```

Comment:
At that point in model spec you should test relation that article indeed belongs to category. Simply checking for db_column is not sufficient.

Reply:
2 more test have been added